TSG/LCM/Add-Node-Failed-at-SetTrustedHostsOnNewNode-Step.md

```diff
@@ -0,0 +1,165 @@
+# How to Resume Add Node operation on BrownField Environment failed at SetTrustedHostsOnNewNode Step
+# Description 
+
+This article describes how to resume Add node operation on a brownfield upgraded cluster when failed at SetTrustedHostsOnNewNode.
+# Symptoms
+ScaleOutOperation action plan for Add node operation will fail with below message at SetTrustedHostsOnNewNode step.
+```
+Connecting to remote server <host-name> failed with the following error message : Access is denied.
+```
+# Mitigation 
+### Apply below mitigation only if PowerShell command ```((Get-StampInformation).InstallationMethod)``` on any clustered host returns **Upgrade** <br/>
```

Comment:
can we create a script block that runs mitigation only when we clustered host returns upgrade so that customer is not confused/applies mitigation in wrong context?

Reply:
this command is only available in clustered hosts as new node is not part of cluster when this failure occurs. Mitigation has steps that needs to run on new node as well as existing nodes in cluster. I can wrap the commands that need to run on clustered node with this check.